src/Cursor.js

```diff
@@ -69,6 +70,15 @@ var cursorBuildMemoizer = util.memoizeFactory(function (cmp, path, value) {
 var build = cursorBuildMemoizer(function (cmp, path, value) {
   path = path === undefined ? [] : path;
   value = value || util.getRefAtPath(cmp.state, path);
+
+  if (_.isObject(value)) {
+    Object.observe(value, function (changes) {
```

Comment:
According to MDN document, only the newest Chrome and Opera supprt this function. It could better to add `typeof Object.observe === 'function'` to `if` statement.